charts/am-org-role-mapping-service/values.preview.template.yaml

```diff
@@ -34,7 +34,8 @@ java:
   environment:
     IDAM_S2S_URL: http://rpe-service-auth-provider-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
     S2S_URL: http://rpe-service-auth-provider-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
-    ROLE_ASSIGNMENT_APP_URL: http://am-role-assignment-service-aat.service.core-compute-aat.internal
+    # AM-3004 ROLE_ASSIGNMENT_APP_URL: http://am-role-assignment-service-aat.service.core-compute-aat.internal
+    ROLE_ASSIGNMENT_APP_URL: https://am-role-assignment-service-pr-2091.preview.platform.hmcts.net
```

Comment:
NB: This temporary change must be reverted before merging.

Reply:
Reverted